olp-cpp-sdk-dataservice-read/src/repositories/NamedMutex.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2020 HERE Europe B.V.
+ * Copyright (C) 2020-2001 HERE Europe B.V.
```

Comment:
Back to the future :)

Reply:
Thanks! Will be updated in the next revision